clients/js/src/cipher.ts

```diff
@@ -213,10 +214,11 @@ export abstract class Cipher {
 export class X25519DeoxysII extends Cipher {
   public override readonly kind = CipherKind.X25519DeoxysII;
   public override readonly publicKey: Uint8Array;
+  public override readonly ephemeralKey: Uint8Array;
   public override readonly epoch: number | undefined;
 
   private cipher: deoxysii.AEAD;
-  private key: Uint8Array; // Stored for curious users.
+  public secretKey: Uint8Array; // Stored for curious users.
```

Comment:
That's not a key we need, added `ephemeralkey`.

Reply:
Derived shared secret key from ephemeral and peer public key. I am not sure what are all use cases looking at comment, but it's not a key needed to init cipher via `X25519DeoxysII.fromSecretKey`